ppr-ui/src/composables/mhrInformation/useMhrUnitNote.ts

```diff
@@ -114,6 +114,17 @@ export const useMhrUnitNote = () => {
 
   const isCancelUnitNote = computed((): boolean => getMhrUnitNoteType.value === UnitNoteDocTypes.NOTE_CANCELLATION)
 
+  const isExpiryDatePassed = (note: UnitNoteIF): boolean => {
+    if ((note.documentType === UnitNoteDocTypes.CONTINUED_NOTE_OF_CAUTION ||
+      note.documentType === UnitNoteDocTypes.EXTENSION_TO_NOTICE_OF_CAUTION) &&
+      note.expiryDateTime != null && note.expiryDateTime !== '') {
+      const expiryDate = note.expiryDateTime.substring(0, 10)
+      const today = localTodayDate()
```

Comment:
I actually see another improvement. Instead of creating `today` date for each row in the table, you could just create it once (for example, when composable is initialized) and then use that for comparison.
This `note.expiryDateTime != null && note.expiryDateTime !== ''` could be refactored to -> `!!note.expiryDateTime`

Reply:
Done!